PXB-2552 Review lockless binlog documentation (8.0) (#1133)
percona/percona-xtrabackup: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/working_with_binary_logs.rst

Working with Binary Logs
========================

> The xtrabackup binary integrates with the log_status table. This integration enables xtrabackup to print out the backup's corresponding binary log position, so you can use it to set up new replication replicas or perform point-in-time recovery.

Reviewer: ```suggestion
The ``xtrabackup`` binary integrates with the ``log_status table``. This integration enables ``xtrabackup`` to print out the backup's corresponding binary log position, so you can use it to set up new replication replicas or perform point-in-time recovery.
```

Author: Fixed.

Finding the Binary Log Position
--------------------------------

> You can find the binary log position corresponding to a backup once the backup has been taken. If your backup is from a server with binary logging enabled, xtrabackup creates a file named xtrabackup_binlog_info in the target directory. This file contains the binary log file name and position of the exact point in the binary log to which the prepared backup corresponds.

Reviewer: ```suggestion
You can find the binary log position corresponding to a backup once the backup has been taken. If your backup is from a server with binary logging enabled, ``xtrabackup`` creates a file named :file:`xtrabackup_binlog_info` in the target directory. This file contains the binary log file name and position of the exact point in the binary log to which the prepared backup corresponds.
```

Author: We are moving away from :file: - otherwise fixed.

You will also see output similar to the following during the backup stage:

.. sourcecode:: text

    210715 14:14:59 Backup created in directory '/tmp/backup/'
    MySQL binlog position: filename 'binlog.000002', position '156'
    . . .
    210715 14:15:00 completed OK!
 
.. note::

  As of Percona XtraBackup 8.0.26-18.0, xtrabackup no longer creates the xtrabackup_binlog_pos_innodb file because MySQL/Percona Server no longer updates the binary log information on global transaction system section of ``ibdata``. Users should rely on ``xtrabackup_binlog_info`` regardless of the storage engine in use.

> The message about hacking group commit refers to an early implementation of emulated group commit in |Percona Server|.

Reviewer: ```suggestion

```

remove

Author: Fixed.

Working with Binary Logs
========================

The ``xtrabackup`` binary integrates with the ``log_status table``. This integration enables ``xtrabackup`` to print out the backup's corresponding binary log position, so that you can use it to set up new replication replicas or perform point-in-time recovery.

Finding the Binary Log Position
--------------------------------

> You can find the binary log position corresponding to a backup after the backup has been taken. If your backup is from a server with binary logging enabled, ``xtrabackup`` creates a file named ``xtrabackup_binlog_info`` in the target directory. This file contains the binary log file name and position of the exact point in the binary log to which the prepared backup corresponds.

Reviewer: ```suggestion
You can find the binary log position corresponding to a backup after the backup has been taken. If your backup is from a server with binary logging enabled, ``xtrabackup`` creates a file named ``xtrabackup_binlog_info`` in the target directory. This file contains the binary log file name and position of the exact point in the binary log to perform point-in-time recovery or provision a new replica.
```

Author: Added.

The output is similar to the following during the backup stage:

.. sourcecode:: text

>     210715 14:14:59 Backup created in directory '/tmp/backup/'

Reviewer: ```suggestion
    210715 14:14:59 Backup created in directory '/backup/'
```

Author: Fixed.
